.travis.yml

```diff
@@ -54,10 +54,11 @@ jobs:
     - stage: Quality
       php: 7.2
       env: PHPStan
+      before_install:
+        - travis_retry composer require --dev --no-update friendsofphp/php-cs-fixer:^2.10 phpstan/phpstan:^0.9
       install: travis_retry composer update --prefer-dist --prefer-stable
-      before_script:
-        - travis_retry composer require --dev --prefer-dist --prefer-stable phpstan/phpstan:^0.9
```

Comment:
running `composer update` and `composer require {package}` was causing composer to create deps graph and conflicts twice, with `composer require --no-update {package}`, it only modify json file, and then resolve all deps/conflicts only one in `composer update` run

TIL: no change in logic, only performance grooming